packages/lesswrong/server/scripts/makeMigrations.ts

```diff
@@ -161,28 +140,80 @@ export const makeMigrations = async ({
 
   for (const collectionName of collectionNames) {
     try {
-      const sql = getCreateTableQueryForCollection(collectionName, forumType);
-
+      const collection = getCollection(collectionName as any);
+      if (!collection) {
+        throw new Error(`Invalid collection: ${collectionName}`);
+      }
+
+      const table = Table.fromCollection(collection, forumType);
+      const createTableQuery = new CreateTableQuery(table);
+      const compiled = createTableQuery.compile();
+      if (compiled.args.length) {
+        throw new Error(`Unexpected arguments: ${compiled.args}`);
+      }
+
+      const sql = compiled.sql + ";";
       const hash = md5(sql);
       currentHashes[collectionName] = hash;
-      
+
       // Include the hash of every collection to make it easier to see what changed
-      schemaFileContents += `-- Schema for "${collectionName}", hash: ${hash}\n`
+      schemaFileContents += `-- Schema for "${collectionName}", hash: ${hash}\n`;
       schemaFileContents += `${format(sql)}`;
+
+      const indexes = table.getIndexes();
+      for (const index of indexes) {
+        const indexName = index.getName();
+        const indexQuery = new CreateIndexQuery(table, index);
+        const indexCompiled = indexQuery.compile();
+        const indexSql = sqlInterpolateArgs(indexCompiled.sql, indexCompiled.args) + ";";
+        const indexHash = md5(indexSql);
+        currentHashes[indexName] = indexHash;
+
+        schemaFileContents += `-- Index "${indexName}" ON "${collectionName}", hash: ${indexHash}\n`;
+        schemaFileContents += `${format(indexSql)}`;
+      }
     } catch (e) {
       console.error(`Failed to check schema for collection ${collectionName}`);
       failed.push(collectionName);
       console.error(e);
     }
   }
 
+  for (const index of expectedCustomPgIndexes) {
+    const trimmed = index.trim().replace(/\s+CONCURRENTLY\s+/gi, " ");
```

Comment:
Indexes can't be created `CONCURENTLY` inside a transaction, but `accepted_schema` is only used to setup new databases from scratch so we should never need to run anything concurrently anyway.